src/main/java/com/kosherjava/zmanim/hebrewcalendar/JewishDate.java

```diff
@@ -1306,7 +1306,7 @@ public void forward(int field, int amount) {
 			}
 		} else if (field == Calendar.MONTH) {
 			forwardJewishMonth(amount);
-		} else if (field == Calendar.YEAR) {
+		} else {
 			setJewishYear(getJewishYear() + amount);
 		}
```

Comment:
Since we throw an exception above if the field is not Calendar.YEAR, Calendar.MONTH, or Calendar.DATE. There is no need to check for the field after checking for two other cases